### DISCUSSION TITLE
TiptapCollabProvider last synced

### DISCUSSION THREAD

**Discussion by kart-c** [ueberdosis/tiptap#5748] [General Questions] [answered]
If in a collaborative editor, the current user starts editing the document, is there a way to get the sync status? Something like "syncing" when the doc is being edited, and then "synced" when user stops writing and the data has been saved successfully in the cloud.

**Comment by nperez0111** [comment, marked as answer]
On the  Hocuspocus/TiptapCollab provider there should be a `hasUnsyncedChanges` which indicates whether there are changes that are not yet synced. You could either poll this field or try to read it on an event like `onMessage` or whenever your editor makes changes
